ui/raidboss/data/triggers/o6s.js

```diff
@@ -1,35 +1,179 @@
+      'Demonic Pain': 'Douleur Démoniaque',
+      'Release': 'Libération',
+      'Easterlies': 'Rafale Ultime',
+      'Rock Hard': 'Brise-roc',
+      'Featherlance': 'Lance De Plume',
+      'Last Kiss': 'Baiser Fatal',
+      'The Price': 'Tentation Mortelle',
+      'Demonic Storm': 'Tempête Démoniaque',
+      'Earthquake': 'Tremblement De Terre',
+      'Flash Torrent': 'Pluie Subite',
+      'Materialize': 'Matérialisation',
+      'Lullaby': 'Berceuse',
+      'Demonic Wave': 'Vague Démoniaque',
+      'Demonic Spout': 'Jaillissement Démoniaque',
+      'Flash Flood': 'Pluie Subite',
+      'Enrage': 'Enrage',
+    },
+    replaceSync: {
+      "I have claimed the girl in the picture! She's mine! You can't have her!": "Héhéhé... La fille du tableau m'appartient. Je ne vous laisserai pas l'avoir !", 
+		  'Demon Chadarnook': 'Démon Chadarnouk',
+	  	'Goddess Chadarnook': 'Déesse Chadarnouk',
+      'Portrayal of Fire': 'Peinture Du Feu',
+      'Portrayal of Wind': 'Peinture Du Vent',
+      'Portrayal of Earth': 'Peinture De La Terre',
+      'Portrayal of Water': "Peinture De L'eau",
+      'Easterly': 'Rafale Ultime',
+      'Haunt': 'Ombre Maléfique',
+    },
+  },
+	{
+    locale: 'jp',
+    replaceText: {
+      'Demonic Howl': 'デモニックハウル',
+      'Demonic Shear': 'デモニックシアー',
+      'Possession': '絵画憑依',
+      'Flash Fire': 'フラッシュファイア',
+      'Flash Gale': 'フラッシュウィンド',
+      'Poltergeist': 'ポルターガイスト',
+      'Divine Lure': '女神の誘惑',
+      'Demonic Pain': 'デモニックペイン',
+      'Release': '憑依解除',
+      'Easterlies': '極風',
+      'Rock Hard': 'ロッククラッシャー',
+      'Featherlance': 'フェザーランス',
+      'Last Kiss': '死の口づけ',
+      'The Price': '死の誘い',
+      'Demonic Storm': 'デモニックストーム',
+      'Earthquake': '地震',
+      'Flash Torrent': 'フラッシュレイン',
+      'Materialize': '実体化',
+      'Lullaby': '子守歌',
+      'Demonic Wave': 'デモニックウェーブ',
+      'Demonic Spout': 'デモニックスパウト',
+      'Flash Flood': 'フラッシュレイン',
+      'Enrage': '怒り',
+    },
+    replaceSync: {
+      "I have claimed the girl in the picture! She's mine! You can't have her!": 'グフフフ……この絵の女は、わしがいただいた……。そう簡単には、返さないぜ……。',
+		  'Demon Chadarnook': 'チャダルヌーク・デーモン',
+		  'Goddess Chadarnook': 'チャダルヌーク・ゴッデス',
+		  'Portrayal of Fire': '火の絵画',
+		  'Portrayal of Wind': '風の絵画',
+		  'Portrayal of Earth': '土の絵画',
+		  'Portrayal of Water': '水の絵画',
+		  'Easterly': '極風',
+		  'Haunt': '悪霊の影',
+    },
+  },
+  ],
   triggers: [
     {
       id: 'O6S Demonic Shear',
-      regex: / 14:2829:Demon Chadarnook starts using Demonic Shear on (\y{Name})/,
+      regex: / 14:2829:(?:チャダルヌーク・デーモン|Demon Chadarnook|Gefallener Chadarnook|Démon Chadarnouk) starts using (?:デモニックシアー|Demonic Shear|Dämonische Schere|Cisailles Démoniaques) on (\y{Name})/,
+
       alertText: function(data, matches) {
         if (matches[1] == data.me)
-          return 'Tank Buster on YOU';
+          return {
+			      en: 'Tank Buster on YOU',
+			      de: 'Tank Buster auf DIR',
+		      };
```

Comment:
Please indent the closing curly brace to line up with the return as it is elsewhere.